en/docs/administer/configure-enterprise-login.md

```diff
@@ -107,31 +107,27 @@ Before you proceed with the configuration, make sure you complete the following:
     3. Click **New Attribute** and select the **Groups** attribute.
     4. Click **Save** and then click **Save Changes**.
 
-### Step 2: Map Asgardeo groups to Choreo roles in the Choreo Console 
+### Step 2: Map Choreo groups to Asgardeo in the Choreo Console 
```

Comment:
This should be `Map Choreo groups to enterprise IdP groups in the Choreo Console`. It's not the Asgardeo groups but the enterprise groups that we need to add the mappings.

Reply:
Done